src/app/pages/datasets/components/dataset-management/dataset-management.component.ts

```diff
@@ -26,7 +26,6 @@ export class DatasetsManagementComponent implements OnInit {
   treeControl = new NestedTreeControl<Dataset, string>((dataset) => dataset.children, {
     trackBy: (dataset) => dataset.id,
   });
-  readonly trackByFn: TrackByFunction<Dataset> = (_, dataset) => dataset.id;
```

Comment:
?

Reply:
The idea of 'trackBy' is to reinitialize the items need in the list and avoid reinitializing items don't need. Just because an item has a unique ID doesn't mean it's suitable for use in the `trackBy` function. it's just to tell the angle when an item needs to be re-rendered, i.e. its lifecycle hook needs to be re-executed. we need to be able to change the content even if Id  reamains the same, so I removed it.